docs/source/azure/examples/resourcegroupsdelayedoperation-general.rst

```diff
@@ -1,7 +1,7 @@
 .. _azure_example_delayedoperation:
 
-Delayed Operations
-==================
+Resource Groups - Delayed operations
```

Comment:
might be worth pointing out that you can construct multi-step workflows using chained marked-for-ops/delayed-ops. probably to defer out of this pr into a separate one on that topic, just fyi.

Reply:
Yeah, we'll have separate 'Scenarios' section (Andy has a PR for logic apps). Good idea to have delayed-op scenarios.